runtime/server/x86/frontend/fft.cc

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2016 HR
```

Comment:
Use fftw3 or fft from PyTorch.

Reply:
Here just keep it, will use a more efficient fft later.